src/components/RenderMdx.js

```diff
@@ -13,12 +13,17 @@ import React from "react"
 
 import { MDXRenderer } from "gatsby-plugin-mdx"
 import { MDXProvider } from "@mdx-js/react"
-import { componentsMapping } from "../components/componentsMapping"
+import { componentsMapping } from "./componentsMapping"
 
-const RenderMdx = ({ children }) => (
-  <MDXProvider components={componentsMapping}>
-    <MDXRenderer>{children}</MDXRenderer>
-  </MDXProvider>
-)
+const RenderMdx = ({ children, overrides }) => {
+  const finalComponentsMapping = overrides
+    ? {...componentsMapping, ...overrides} : componentsMapping
+
+  return (
+    <MDXProvider components={finalComponentsMapping}>
```

Comment:
I'm going to be pedantic here but this can just become:

```javascript
const RenderMdx = ({ children, overrides = {} }) => {
  return (
    <MDXProvider components={{...componentsMapping, ...overrides} }>
```

By providing a default value for `overrides` as the empty object you can just avoid that ternary.

Reply:
not pedantic, totally fair. I will address this in my next PR. Thanks for the review!